server/bot/commands/trivia.ts

```diff
@@ -0,0 +1,259 @@
+import { Message, GuildTextableChannel, TextableChannel, User, EmbedOptions, Client } from 'eris'
+import { Command, DB } from '../imports/types'
+import { getInsult } from '../imports/tools'
+import fs from 'fs'
+
+async function parseTriviaList (fileName: string) {
+  const data = await fs.promises.readFile(`./triviaLists/${fileName}.txt`, 'utf8')
+  const triviaList = new Map<string, string[]>()
+  data.split('\n').forEach(el => {
+    const splitEl = el.split('`').map(ans => ans.trim()).filter(ans => !!ans)
+    if (splitEl.length >= 2) triviaList.set(splitEl[0], splitEl.slice(1))
+  })
+  return triviaList
+}
+
+export class TriviaSession {
+  settings: { maxScore: number, timeout: number, timeLimit: number, botPlays: boolean, revealAnswer: boolean }
+  currentLine: {question: string, answers: string[]}
+  channel: TextableChannel
+  author: User
+  message: Message
+  questionList: {question: string, answers: string[]}[]
+  scores: { [id: string]: number } = {}
```

Comment:
This breaks if there's 2 people with the same username, use user ID instead and store username#discriminator and score on an object on it. OR use username#discriminator, but this can change mid-game, rare but possible, which would break it too. I think just use `member.guild.members` cache to retrieve username#discriminator or whatever (Eris will update this cache when updates come in) or to use a mention (might be annoying though, or might not work in embeds).